### PR TITLE
feat: supported onprem

### PR DESCRIPTION
**Pull request by ye11ow**
Removed replicas since proton now provides default value

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "Timeplus Provider"
+page_title: "timeplus Provider"
```
Comment (jovezhong) [inline docs/index.md]: Pls keep `Timeplus`

File: docs/index.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 subcategory: ""
 description: |-
   The Timeplus provider is used to interact with the resources supported by Timeplus https://www.timeplus.com/ in a workspace. The provider needs to be configured with an API key before it can be used.
   Use the navigation to the left to read about the available resources.
 ---
 
-# Timeplus Provider
+# timeplus Provider
```
Comment (jovezhong) [inline docs/index.md]: No need to change

File: docs/index.md [hunk 3]
```diff
@@ -36,9 +36,11 @@ provider "timeplus" {
 
 ### Required
 
-- `api_key` (String, Sensitive) The API key to be used to call Timeplus API.
 - `workspace` (String) The ID of the workspace in which the provider manages resources.
 
 ### Optional
 
+- `api_key` (String, Sensitive) [Cloud] The API key to be used to call Timeplus Enterprise Cloud.
 - `endpoint` (String) The base URL endpoint for connecting to the Timeplus workspace. When it's not set, `https://us.timeplus.cloud` will be used.
```
Comment (jovezhong) [inline docs/index.md]: we should change default value to us-west-2

File: docs/index.md (hunk 2 above)
Comment (ye11ow) [reply docs/index.md]: it is auto-generated

File: docs/index.md (hunk 3 above)
Comment (ye11ow) [reply docs/index.md]: updated

File: docs/index.md (hunk 2 above)
Comment (ye11ow) [reply docs/index.md]: updated

**Pull request merged**